### PR TITLE
Feature/update alerts

### PR DESCRIPTION
**Pull request by PhungNg**
## Description
Update the Alert component and documentation

## How Has This Been Tested?
Ran and updated existing tests.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/App/ComponentsDocumentation/components/Alerts/constants.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    id: "overviewAlert",
+    tabsId: "overviewAlertTabs",
+    elements: [
+        {
+            tab: "Error",
+            component: <Alert type="danger" icon="cancel" text={exampleText} />,
+            options: {
+                checkbox: [
+                    {
+                        title: "Add",
```
Comment (eskilsen) [inline src/App/ComponentsDocumentation/components/Alerts/constants.js:16]: Please change to "Options" for keeping consistency. 💯 
Great work daddy cool!

File: src/App/ComponentsDocumentation/components/Alerts/index.js [hunk 2]
```diff
@@ -1,103 +1,111 @@
...
+    </section>
 );
 
-const JavascriptMethods = () => (
-    <>
-        <h2 id="javascript-methods">JavaScript methods</h2>
-        <JavascriptDocs componentName="alert" />
-    </>
+const DeveloperDocumentation = () => (
+    <section>
```
Comment (eskilsen) [inline src/App/ComponentsDocumentation/components/Alerts/index.js:83]: Add usage with npm section

File: src/App/docutils/ComponentPreview/index.js [hunk 3]
```diff
@@ -174,6 +174,7 @@ const ComponentPreview = ({ children, language, removeOuterTag, hideValue, remov
 
             this.state = {
                 activeTab: this.props.showCasePanelAdvanced.elements[0],
+                hideOptions: this.props.showCasePanelAdvanced.hideOptions,
                 optionsOpen: window.innerWidth > 1200, // XL grid breakpoint
                 activeOptions: this.props.showCasePanelAdvanced.elements[0].activeOptions ? [...this.props.showCasePanelAdvanced.elements[0].activeOptions] : []
             };
```
Comment (eskilsen) [inline src/App/docutils/ComponentPreview/index.js:180]: Remove from this branch 🦜

**Review by eskilsen — CHANGES_REQUESTED** [review]
Great job! Look at my comments and then ask for new review 🥇

File: src/App/ComponentsDocumentation/components/Alerts/constants.js (hunk 1 above)
Comment (PhungNg) [reply src/App/ComponentsDocumentation/components/Alerts/constants.js:16]: 😂

**Review by eskilsen — APPROVED** [review]
I approve this message 💯

**Pull request merged**